src/lesson8/geometric_figures/Shape.java

```diff
@@ -0,0 +1,12 @@
+package lesson8.geometric_figures;
+
+public abstract class Shape implements AreaDetermination {
+
+
+    @Override
+    public double calcArea() {
+        return 0;
+    }
```

Comment:
you need to use the interface AreaDetermination in children of Shape class, Shape is abstract and I don't see a point of using it here (AreaDetermination)